pkg/onmetal/create_machine_test.go

```diff
@@ -141,12 +137,16 @@ var _ = Describe("CreateMachine", func() {
 				Name:      fmt.Sprintf("%s-ignition", machineName),
 			},
 		}
+
 		Eventually(Object(ignition)).Should(SatisfyAll(
-			HaveField("Data", map[string][]byte{
-				"ignition.json": SampleIgnition,
-			}),
+			HaveField("Data", HaveKey("ignition.json")),
 		))
 
+		ignitionJSONData := ignition.Data["ignition.json"]
```

Comment:
Move this part up as you need this later.

Reply:
Done